[2024-09-22T12:08:15.443+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T12:08:15.460+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-09-22T12:07:06.908148+00:00 [queued]>
[2024-09-22T12:08:15.467+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-09-22T12:07:06.908148+00:00 [queued]>
[2024-09-22T12:08:15.468+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-22T12:08:15.477+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2024-09-22 12:07:06.908148+00:00
[2024-09-22T12:08:15.483+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1961) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T12:08:15.484+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'manual__2024-09-22T12:07:06.908148+00:00', '--job-id', '699', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp8klmu5i_']
[2024-09-22T12:08:15.485+0000] {standard_task_runner.py:72} INFO - Started process 1962 to run task
[2024-09-22T12:08:15.486+0000] {standard_task_runner.py:105} INFO - Job 699: Subtask load
[2024-09-22T12:08:15.538+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load manual__2024-09-22T12:07:06.908148+00:00 [running]> on host 6e2558f7c62d
[2024-09-22T12:08:15.622+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T12:07:06.908148+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T12:07:06.908148+00:00'
[2024-09-22T12:08:15.624+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T12:15:59.178+0000] {local_task_job_runner.py:318} WARNING - Recorded pid 2079 does not match the current pid 1962
[2024-09-22T12:15:59.185+0000] {local_task_job_runner.py:218} WARNING - Heartbeat failed with Exception: PID of job runner does not match
[2024-09-22T12:16:04.329+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2024-09-22T12:16:04.338+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-22T12:16:04.343+0000] {process_utils.py:132} INFO - Sending 15 to group 1962. PIDs of all processes in the group: [1962]
[2024-09-22T12:16:04.345+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1962
[2024-09-22T12:17:04.348+0000] {process_utils.py:150} WARNING - process psutil.Process(pid=1962, name='airflow task ru', status='sleeping', started='12:08:15') did not respond to SIGTERM. Trying SIGKILL
[2024-09-22T12:17:04.377+0000] {process_utils.py:87} INFO - Sending the signal 9 to group 1962
[2024-09-22T12:17:04.397+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1962, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='12:08:15') (1962) terminated with exit code -9
[2024-09-22T12:17:04.400+0000] {standard_task_runner.py:190} ERROR - ('Job 705 was killed before it finished (likely due to running out of memory)', 'For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed')
